docs/_workshops/object-detection/Annotations to Colab.md

```diff
@@ -0,0 +1,36 @@
+
+---
+title: Training a Model
+order: 6
+redirect_from:
+  - /object-detection/cli/6.html
+---
+
+Cloud Annotations focuses on the dataset creation aspect of the model development lifecycle leaving the training up to you. There are many ways to train your model, each with their own use cases and tradeoffs. You could train from scratch using a framework like [TensorFlow](https://www.tensorflow.org/) or [PyTorch](https://pytorch.org/), use a drag & drop tool like Apple's [Create ML](https://developer.apple.com/machine-learning/create-ml/) or use a cloud managed solution like [Watson Machine Learning](https://www.ibm.com/cloud/machine-learning).
+
+Today we'll be using an online tool called Google Colab as a free and easy way to get our first model trained.
+
+## Prepare to Export to Colab
+
+#### Training with Google Colab
```

Comment:
delete `## Prepare to Export to Colab`
and change the other title to h2 `## Training with Google Colab`